src/templates/article.js

```diff
@@ -78,29 +81,58 @@ const ArticleContent = styled('article')`
     max-width: ${size.maxContentWidth};
     padding-left: ${size.small};
     padding-right: ${size.small};
+
+    ${StyledContainerTop} {
+        display: flex;
+        justify-content: flex-end;
+        margin-bottom: 48px;
+    }
+
     @media ${screenSize.upToLarge} {
         margin: 0 auto;
+
+        ${StyledContainerTop} {
+            display: none;
+        }
     }
 `;
 const Icons = styled('div')`
+    display: flex;
     margin: ${size.tiny} ${size.default};
     span {
         padding: 0 ${size.tiny};
     }
+
     @media ${screenSize.largeAndUp} {
-        display: flex;
         flex-direction: column;
+        margin-top: 80px;
```

Comment:
This seems a bit hacky to me. Could we have the `ArticleRating` outside of `Icons` and then below have content as normal (aligned next to each other)? Or, we can move outside of `Container` as then the flex won't apply.

CSS Grid might be a good approach here because we can easily change the grid layout for mobile by using something like `grid-template-areas`. Check out this link https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas